package.json

```diff
@@ -11,14 +11,16 @@
     "build:lib": "babel --out-dir lib src",
     "prebuild:umd": "rm -rf dist/*",
     "prebuild:dist": "rm -rf dist/*",
+    "build:dev": "rollup -c",
```

Comment:
yeah, I agree. I think output `styled-components.js` and `styled-components.min.js` is cost more time. so I want just output one file. I will change it.